docker/opensource.md

```diff
@@ -87,5 +87,102 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # phpunit
 ```
 
+## Cosmo-Tech
+{id: docker-python-Cosmo-Tech}
+
+
+* [Cosmo-Tech](https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library)
+
+```
+$ git clone https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library.git
+$ cd CosmoTech-Acceleration-Library
+$ docker run -t -i --name cosmotech-acceleration-library-dev -v /opt/CosmoTech-Acceleration-Library -v /C:/Users/YourUserName/CosmoTech-Acceleration-Library:/opt python:3.11 bash
```

Comment:
Why are there two -v parameters? The 2nd one is for Windows, right?

Reply:
Maybe is realted to that I run it on Git Bash?

I tried with w but get erroes, and try to dix it but it didnt work:
docker run -it --name cosmotech-acceleration-library-dev -w /opt -v /C:/Users/Ran/CosmoTech-Acceleration-Library:/opt python:3.11 bash               
docker: Error response from daemon: the working directory 'C:/Program Files/Git/opt' is invalid, it needs to be an absolute path.
See 'docker run --help'.

docker run -it --name cosmotech-acceleration-library-dev -w /opt -v %cd%:/opt python:3.11 bash
docker: Error response from daemon: the working directory 'C:/Program Files/Git/opt' is invalid, it needs to be an absolute path.
See 'docker run --help'.

Reply:
And I'm quite confused, what is meant to do more for Linux? I was working with Windows, but I ran everything through Git Bash and it worked fine for me with no problems.

Reply:
I could not run docker from git bash properly either. Only CMD and PowerShell